source/change-streams/tests/unified/change-streams.yml

```diff
@@ -334,6 +334,10 @@ tests:
           newField: "newFieldValue"
 
   - description: "Test new structure in ns document MUST NOT err"
+    runOnRequirements:
+      - minServerVersion: "3.6"
+        maxServerVersion: "5.2"
```

Comment:
To be clear, this test starts to fail on 5.3 as the server fix hasn't been backported to that version? I suppose this could be `maxServerVersion: 5.2.99` but it doesn't really matter.

Reply:
Correct, the server team hasn't yet decided whether to prioritize the backport fix.  That's a fair point about the version, let's do that.